app/controllers/issues_controller.rb

```diff
@@ -94,6 +94,47 @@ def import
     @query = params[:query]
   end
 
+  def merging
+    @issues = Issue.where(id: params[:sources])
+
+    if @issues.count <= 1
+      redirect_to issues_url
+    end
+  end
+
+  def merge
+    count = 0
+    if params[:sources]
+
+      # create new issue if existing issue not given
+      if @issue.new_record?
+        @issue.author ||= current_user.email
+        if @issue.save && @issue.update_attributes(issue_params)
+          track_created(@issue)
+          tag_issue_from_field_content(@issue)
+        end
+      end
+
+      count = 0
+      if @issue.persisted?
+        source_ids = params[:sources].map(&:to_i) - [@issue.id]
+        count = @issue.merge source_ids
+      end
+
+    end
+
+    respond_to do |format|
+      format.html {
+        if count > 0
+          redirect_to issues_url, notice: "#{count} issues merged into #{@issue.title}."
```

Comment:
I think it would be better if, after merging, the user gets redirected to the new issue.